### ISSUE TITLE
React Devtools still active in a production build.

### ISSUE THREAD

**Issue by crucialfelix** [firejune/electron-react-devtools#3]
Thanks for the package, I find it essential for development !

I am using electron-prebuilt 1.2.3. Maybe I can move to the upstream react-devtools now ?

The problem:

For some reason when I make a production release, electron-react-devtools is still activating and there is still a React tab. 

But it fails and throws an error because the devDependencies are not included with the production build and so `react-developer-tools` fails to import portfinder.

`module.js:440 Uncaught Error: Cannot find module 'portfinder'`

This is somehow being injected by electron-react-devtools, but to be honest I can't understand why.

![screen shot 2016-06-22 at 17 32 57](https://cloud.githubusercontent.com/assets/218738/16272828/98f127b6-389f-11e6-8248-1213a1269cad.png)

I installed as per instructions with `require('electron-react-devtools').install();`

Where did it install to ? How do I uninstall it ? I flushed the electron-prebuilt dist cache, but it is still there.

I do NOT do this anymore:

```
// in development.
  if (process.env.NODE_ENV !== 'production') {
    require('electron-react-devtools').inject()
  }
```

as I am on 1.2.3. So I am not even requireing it at all and definitely not injecting.

Thank you for your help !

**Comment by crucialfelix** [comment]
I think I found it: `/Users/crucial/Library/Application Support/PlaySPLOM/`

This is for the Electron app I am working on.

There is a Preferences file in there and in some applications there is also a `DevTools Extensions`

That is perhaps how it kept injecting itself.

Even though I removed electron-react-devtools from devDependencies, it would still somehow activate. I finally had to also remove the source from node_modules.
